providers/tokenbucket/limiter.go

```diff
@@ -0,0 +1,29 @@
+package tokenbucket
+
+// Implement Limiter interface
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/juju/ratelimit"
+)
+
+// TockenBucketInterceptor implement tocken bucket algorithm
```

Comment:
Let's write comments with full sentences :hug: (trailing dot and capital letter first word)